command/agent/exec/exec.go

```diff
@@ -333,3 +359,14 @@ func (s *Server) restartChildProcess(newEnvVars []string) error {
 
 	return nil
 }
+
+func (s *Server) Close() {
+	s.childProcessLock.Lock()
+	defer s.childProcessLock.Unlock()
+	s.close()
+}
+
+func (s *Server) close() {
+	_ = s.childProcessStdout.Close()
+	_ = s.childProcessStderr.Close()
```

Comment:
Would this potentially close the global os.Stdout and os.Stderr? https://go.dev/play/p/ksHYajR4CVU